client/src/pages/Home/HeroBanner.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+
+import { Button } from '@/components/ui/button';
+import { TypewriterEffect } from '@/components/ui/typewriter-effect';
+import heroImg from '../../assets/HeroImage.jpg';
+import { FaCheckCircle } from 'react-icons/fa';
+import { FaChevronRight } from 'react-icons/fa';
+
+interface chatBoxComp {
+  text: string;
+  bgColor: string;
+  textColor: string;
+  borderRadius: string;
+  top?: string;
+  bottom?: string;
+  left?: string;
+  right?: string;
+}
+
+const ChatBox: React.FC<chatBoxComp> = ({
```

Comment:
Try to use functional declarations for components and functional expressions for utility functions - good coding practices :)